OpenSim/Common/Component.h

```diff
@@ -2463,19 +2465,20 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
         bool hidden;
     };
 
-
     /// Helper method to enable Component makers to specify the order of
     /// subcomponents to be added to the System used by addToSystem().
-    /// Use this method in extendConnect() of your Component to set the
-    /// order of your components. For example, Model orders subcomponents
-    /// according to the multibody tree and add bodies and joints in order
+    /// Use this method in extendConnect() of your Component (or within 
+    /// your extendConnectToModel() for ModelComponents) to set the
+    /// order of your subcomponents. For example, Model orders subcomponents
+    /// according to the multibody tree and adds bodies and joints in order
     /// starting from Ground and growing outward.
-    /// Note: call resetSubcomponentOrder() prior to setting the first
-    /// subcomponent.
     /// If the subcomponent already appears in the order list setting it
     /// later in the list has no effect. The list remains unique.
```

Comment:
Can you add a comment here to the effect of "You should *rarely* need to use this; it is mainly used by Model."